### DISCUSSION TITLE
一直在申请订阅

### DISCUSSION THREAD

**Discussion by fanqieD** [vernesong/OpenClash#2448] [Q&A]
N1旁路由，昨天刚配置好，今天早上0点到2点左右，累积申请订阅44次，直到机场给我暂时禁用了，有遇到过相同问题的朋友吗

**Comment by CT702-7** [comment]
同 兄弟找到解决这个问题的方法了吗

**Reply by fanqieD** [reply]
貌似是因为当时我浏览器一直开着openclash的主界面，会导致10s刷新一次

**Reply by CT702-7** [reply]
我一直在想能不能固定使用一个节点来订阅...
我当时猜这可能是一个原因，但一直没试。回头把自动刷新关了试试😂

**Comment by vernesong** [comment]
半小时刷新一次订阅信息，有问题请更新下
